api/src/main/resources/marquez/db/migration/V67.1__job_versions_io_mapping_add_job_reference.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE job_versions_io_mapping ADD COLUMN job_uuid uuid REFERENCES jobs(uuid) ON DELETE CASCADE;
+ALTER TABLE job_versions_io_mapping ADD COLUMN symlink_target_job_uuid uuid REFERENCES jobs(uuid) ON DELETE CASCADE;
```

Comment:
Minor: `symlink_target_job_uuid` -> `job_symlink_target_uuid` since it's defined in the `jobs` table as `jobs.symlink_target_uuid`